cmd/ai/sre_agent/validate_cluster.go

```
@@ -0,0 +1,126 @@
+		SilenceUsage:  true,
+		SilenceErrors: true,
+		Run: func(cmd *cobra.Command, args []string) {
+			if err := validateCluster(clusterID); err != nil {
+				fmt.Fprintf(os.Stderr, "Error: %v\n", err)
+				os.Exit(1)
+			}
+		},
+	}
+
+	validateClusterCmd.Flags().StringVarP(&clusterID, "cluster-id", "C", "", "Cluster ID (internal or external)")
+	validateClusterCmd.MarkFlagRequired("cluster-id")
```

golangci-lint (2.12.2)

[error] Error return value of `validateClusterCmd.MarkFlagRequired` is not checked

(errcheck)